Kajabi/sage-lib
# [DSS-247] Docs Site Housekeeping

## Description
Various housekeeping-related updates and bug fixes for the docs site.

1. **Duplicate Pages**
Remove the `/pages/design` route since it's causing duplicates
http://localhost:4000/pages/patterns/grid
http://localhost:4000/pages/design/grid
===
http://localhost:4000/pages/foundations/icon
http://localhost:4000/pages/design/icon

2. **Panel & Cards:**
http://localhost:4000/pages/patterns/panels_cards
Adjustments to move this page to the sidebar under layout tools section.
Other `/patterns` pages are already under layout tools

|  Before  |  After  |
|--------|--------|
|![Screen Shot 2022-12-14 at 2 36 49 PM](https://user-images.githubusercontent.com/1175111/207730143-e9ce0493-c96d-43b1-8bd6-66c615c1a382.png)|![Screen Shot 2022-12-14 at 2 36 34 PM](https://user-images.githubusercontent.com/1175111/207730165-a6cc478b-84cc-464a-8457-2580d57f314b.png)|


3. **Container page:**
http://localhost:4000/pages/interactions/container
Removes legacy page. There are container pages under layout tools and components.

4. **Homepage patterns panel**
Updates card copy to layout tools verbiage
Updates the link to navigate to layout tools

|  Before  |  After  |
|--------|--------|
|![Screen Shot 2022-12-14 at 2 32 31 PM](https://user-images.githubusercontent.com/1175111/207729348-ca1c6d7e-9bf4-4218-8b06-6fc00d55116f.png)|![Screen Shot 2022-12-14 at 2 32 16 PM](https://user-images.githubusercontent.com/1175111/207729397-b5c6ced8-6305-4767-a529-74b81984fdf0.png)|

5. **Minor copy adjustments.**


## Testing in `sage-lib`

- Navigate to the [docs site](http://localhost:4000/pages/index)
- Verify the changes listed above.


## Testing in `kajabi-products`
1. (**LOW**) Docs site housekeeping items. Docs only update, no effect on KP.



## Related
https://kajabi.atlassian.net/browse/DSS-247
